htdocs/core/class/commoninvoice.class.php

```diff
@@ -223,7 +223,8 @@ function getLibType()
 	 */
 	function getLibStatut($mode=0,$alreadypaid=-1)
 	{
-		return $this->LibStatut($this->paye,$this->statut,$mode,$alreadypaid,$this->type);
+		$paiement = $this->getSommePaiement();
```

Comment:
Not required. This value must be provided by caller into the second parameter of getLibStatut